api/src/modules/characters/controllers/character.ctrl.ts

```diff
@@ -0,0 +1,29 @@
+import { AppRequestHandler } from 'express';
+import boom from '@hapi/boom';
+import status from 'http-status-codes';
+
+import { logger } from '../../../shared/logger';
+import { createNewCharacterForUser } from '../characters.context';
+
+interface CreateNewCharacterReq {
+  body: {
+    name: string;
+    race: string;
+  };
+}
+export const createNewCharacter: AppRequestHandler<CreateNewCharacterReq> = async (
+  req,
+  res
+) => {
+  const { name, race } = req.body;
+  const user = req.user;
+
+  try {
+    const character = await createNewCharacterForUser(user, name, race);
+    logger.info(`Character "${name} with race ${character.race}" created`);
+
+    return res.status(status.OK).send({ id: character.id });
+  } catch (err) {
+    throw boom.badRequest(err.message);
```

Comment:
how u know that there can be an bad request error?
if you do not know what kind of error it is, just let it be - it will generate 500 and let us know that something really wrong happen.